docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -259,9 +260,12 @@ lines of output showing that the SGX enclave has been initialized:
     There’s quite a bit going on in the previous ``sawset proposal`` command, so
     let’s take a closer look at what it accomplishes:
 
-    ``sawtooth.consensus.algorithm=poet``
+    ``sawtooth.consensus.algorithm=PoET``
```

Comment:
This should be sawtooth.consensus.algorithm.name

Reply:
Fixed.